tests/pytorch/nightly/resnet50-mp.libsonnet

```diff
@@ -65,7 +65,7 @@ local tpus = import 'templates/tpus.libsonnet';
               FINAL: {
                 fixed_value: {
                   comparison: 'GREATER',
-                  value: 75.0,
+                  value: 74.0,
```

Comment:
Does this change both pod and donut?

Reply:
Yes. Do we want to keep it at 75% for donut?

Reply:
ideally yes.. This way when donut regress we can catch that.

Reply:
Done. Checked the diff and this only affects the pod version of the test.